src/routes/authentication-route.js

```diff
@@ -3,19 +3,31 @@
 const express = require('express');
 const jwt = require('jsonwebtoken');
 const config = require('config');
+const hashPassword = require('../helpers/hash/hash-factory')();
 
 module.exports = function createAuthenticationRoute(repository) {
   const router = express.Router();
 
   async function loginUser(request, response) {
     const result = await repository.returnUser(request.body);
     if (!result) return response.sendStatus(404);
-    const token = jwt.sign(result, config.jwtconf.secret, config.jwtconf.time);
+    const token = jwt.sign({ role: result[0], user: result[1].id }, config.jwtconf.secret, config.jwtconf.time);
```

Comment:
extract the result using `const [role, user] = result` it's much cleaner and reads easier.